src/kind.ts

```diff
@@ -0,0 +1,79 @@
+import { Job } from "@brigadecore/brigadier";
+
+export const kindJobImage = "radumatei/golang-kind:1.11-0.4";
+
+export class KindJob extends Job {
+    constructor(name: string, image?: string) {
+        if (image == undefined) {
+            image = kindJobImage;
+        }
+        super(name, image);
+
+        // kind needs to run as a privileged pod
+        this.privileged = true;
+
+        // since the cluste creation takes some time,
+        // set the timeout to 30 minutes
+        this.timeout = 180000;
```

Comment:
I wonder if we include a note in the added docs mentioning this `timeout` default and how users may override it?